ocp4/profiles/coreos-ncp.profile

```diff
@@ -148,9 +148,9 @@ selections:
     - sysctl_kernel_kexec_load_disabled
     - sysctl_kernel_yama_ptrace_scope
     - sysctl_kernel_perf_event_paranoid
-    - sysctl_user_max_user_namespaces
-    - sysctl_user_max_user_namespaces.role=unscored
-    - sysctl_user_max_user_namespaces.severity=info
+    #- sysctl_user_max_user_namespaces
```

Comment:
I would outright remove the rules or at least added a comment why are the rules disabled